app/models/email_volume/organisation_weekly_email_volume.rb

```diff
@@ -0,0 +1,37 @@
+module EmailVolume
+  class OrganisationWeeklyEmailVolume
+    HIGH = '40 - 60'.freeze
+    MEDIUM = '0 - 20'.freeze
+    LOW = '0 - 5'.freeze
```

Comment:
Are these numbers based on anything?

Reply:
It's very much a finger-in-air estimate. This is the same as our estimates for taxonomy emails (which may now be wrong).

This is the explanation in the commit (https://github.com/alphagov/email-alert-frontend/commit/d4a97b2b345c5f85f731d55075336e9ae70c6bdc#diff-5c90fde72d5dd7ed3acd972c5bc07f58) where the taxonomy estimate was added:

> These figures are based on analysis of historical email volume.

Perhaps we could make a request to email-alert-api to figure out the exact volume.